### DISCUSSION TITLE
Elongated Grains along preferred orientation with periodic boundaries.

### DISCUSSION THREAD

**Discussion by mlucasq** [neperfepx/neper#843] [Q&A] [answered]
Hi, thanks in advance for your help. 

I am getting an error when trying to apply periodicity to a domain with elongated and rotated grains. The figure below shows an example of a domain with elongated grains along a preferred orientation (45 deg). The question is, how can I apply periodic boundaries to such case? Is it possible? 
![N11520_R20x_SS316L_2MHz_45deg_img](https://github.com/neperfepx/neper/assets/105709803/6451f387-cb61-4cfb-9388-9249a5fc83fa)

I have been able to generate a domain with elongated grains and apply periodicity for 0 and 90deg preferred orientation angles, as shown in the figure below for 0 deg. I saw a similar discussion #331 from 2021 for periodicity with columnar grains. However, I thought it could be something already solved in recent updates and would give it shot here. Sorry if this configures a duplicated topic. 
![N11520_R20x_SS316L_2MHz_0deg_img](https://github.com/neperfepx/neper/assets/105709803/b059decb-12c8-4a87-bbdc-d00aa3cdb857)

This is the neper command for 45 deg case leading to the error message "Aborted (core dumped)":
```
Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n 550.0 -morpho voronoi -domain
         square(10,50):rotate(0,0,1,-45):scale(0.2,1.0,1.0) -dim 2 -ori
         fiber(1,0,0,0.7071067811865476,0.7071067811865475,0)
         -oridescriptor euler-bunge:passive -crysym cubic -periodicity y
         
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :   - Setting seeds... 
Info   :   - Generating crystal orientations...
Info   :   - Running tessellation...
Aborted (core dumped)
```

Any help would be greatly appreciated. 
Kind regards,
Lucas

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.

**Reply by mlucasq** [reply]
Hi @rquey . Do you have any updates about the question above? Thanks!
